snuba/datasets/configuration/generic_metrics/storages/sets.yaml

```diff
@@ -81,3 +81,9 @@ allocation_policies:
 query_processors:
   - processor: TableRateLimit
   - processor: TupleUnaliaser
+  - processor: MappingOptimizer
```

Comment:
we don't want to apply this with the migration. Remove this before merging once the distributed test passes